src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs

```diff
@@ -0,0 +1,288 @@
+        public bool Mute
+        {
+            get
+            {
+                NativeWebRTC.GetMute(WebRtc.Handle, SourceId.Value, MediaType, out bool isMuted).
+                    ThrowIfFailed("Failed to get mute");
+
+                return isMuted;
+            }
+            set
+            {
+                NativeWebRTC.SetMute(WebRtc.Handle, SourceId.Value, MediaType, value).
+                    ThrowIfFailed("Failed to set mute");
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the video resolution of the current media source.
+        /// </summary>
+        /// <value>A value that specifies the mute status.</value>
+        /// <exception cref="ArgumentException">This source is not video source.</exception>
+        /// <exception cref="InvalidOperationException">MediaSource is not attached yet.</exception>
+        /// <since_tizen> 9 </since_tizen>
+        public Size VideoResolution
+        {
+            get
+            {
+                if (MediaType != MediaType.Video)
+                {
+                    throw new ArgumentException("This property is only for video.");
+                }
+
+                NativeWebRTC.GetVideoResolution(WebRtc.Handle, SourceId.Value, out int width, out int height).
+                    ThrowIfFailed("Failed to get video resolution");
+
+                return new Size(width, height);
+            }
+            set
+            {
+                if (MediaType != MediaType.Video)
+                {
+                    throw new ArgumentException("This property is only for video.");
+                }
+
+                NativeWebRTC.SetVideoResolution(WebRtc.Handle, SourceId.Value, value.Width, value.Height).
+                    ThrowIfFailed("Failed to set video resolution");
+            }
+        }
+
+        /// <summary>
+        /// Enables the audio loopback. The local audio will be played with <paramref name="policy"/>.
+        /// </summary>
+        /// <param name="policy">The <see cref="AudioStreamPolicy"/> to apply.</param>
+        /// <remarks>
+        /// <see cref="MediaSource"/> does not support all <see cref="AudioStreamType"/>.<br/>
+        /// Supported types are <see cref="AudioStreamType.Media"/>, <see cref="AudioStreamType.Voip"/>,
+        /// <see cref="AudioStreamType.MediaExternalOnly"/>.<br/>
+        /// </remarks>
+        /// <exception cref="ArgumentNullException"><paramref name="policy"/> is null.</exception>
+        /// <exception cref="InvalidOperationException">This MediaSource is not Audio</exception>
+        /// <exception cref="NotSupportedException">
+        ///     <see cref="AudioStreamType"/> of <paramref name="policy"/> is not supported on the current platform.
+        /// </exception>
+        /// <exception cref="ObjectDisposedException">
+        ///     <paramref name="policy"/> has already been disposed.
+        /// </exception>
+        /// <returns>The track ID.</returns>
```

Comment:
the return type is MediaStreamTrack, a little more than just ID, even if the ID is the most important thing here

Reply:
Done